### PR TITLE
Translation of Emulating-Haiku-In-VMware-Fusion.html to Markdown

### PR DESCRIPTION
**Pull request by johngarbi**
For Google Code-In.

### CODE REVIEW

**Comment by waddlesplash** [comment]
No need to move the images. And you've got lots of extraneous stuff in the Markdown...

**Comment by johngarbi** [comment]
Sorry, I'm new to md. What part, specifically, is extreneous?

**Comment by johngarbi** [comment]
The images have been fixed.

File: content/guides/virtualizing/vmware-fusion.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+There are several methods offered to install Haiku. Among those, only the vmdk and the iso methods are suitable for running Haiku easily on VMware Fusion. Basically these are the two methods that will be shown in this guide. The easiest of all is utilising the vmdk which is already properly configured.
+
+> Running Haiku from a VM image is recommended to avoid any possible problems. Unless it does not work, or you would like to simulate an actual installation, do stick to this method.
```
Comment (waddlesplash) [inline content/guides/virtualizing/vmware-fusion.md]: This needs to be an infobox. See the already converted guides or other parts of the site for how to do that.

File: content/guides/virtualizing/vmware-fusion.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+##### Go to section:
+
+*   [Running Haiku from a VM image.](#part_vmimage)
+*   [Installing and running Haiku from an ISO image.](#part_iso)
+*   [Additional Steps.](#part_additional)
+*   [Troubleshooting.](#part_trouble)
+
+### Running Haiku from a VM image.
+
+##### Step 1\. Downloading Haiku.<a name="part_vmimage"></a>
```
Comment (waddlesplash) [inline content/guides/virtualizing/vmware-fusion.md]: Unneccessary \ (I think?). And all the ones further down the page.

File: content/guides/virtualizing/vmware-fusion.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+##### Step 9\. Initialising partition.
+
+Don't worry if you see the warning saying Haiku can't find any partitions to boot from. Click <kbd>OK</kbd> and we will setup the necessary partitions needed to install Haiku. Click <kbd>Set up partitions</kbd>. Before clicking on the device with the hard disk icon. Then click Disk > Initialize > Intel Partition Map…
+
+##### Step 10\. Selecting partition.
+
+Select the new space and click Partition > Create.
+
+> It is important that you tick the active partition or else your disk will not boot!
```
Comment (waddlesplash) [inline content/guides/virtualizing/vmware-fusion.md]: Same infobox deal here.

File: content/guides/virtualizing/vmware-fusion.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+##### Step 12\. Selecting your new partition.
+
+Close the dialog box and select the partition you just created. Ensure that the partition is correct.
+
+##### Step 14\. Installing Haiku.
+
+Begin the installation! Reboot once you are done, you should be able to see the Haiku desktop after you reboot. If you are stuck at the installation screen after reboot, disconnect the iso from the virtual disk drive and try again.
+
+##### Step 14\. You are now finished.
```
Comment (waddlesplash) [inline content/guides/virtualizing/vmware-fusion.md]: Er, shouldn't this be 15?

File: content/guides/virtualizing/vmware-fusion.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+After downloading the ISO image, unzip the file and in the folder, you will see a .iso file. Under Virtual Machine Library, which you can use the shortcut <kbd>Shift</kbd>+<kbd>cmd</kbd>+<kbd>L</kbd>, click Add and then New.
+
+##### Step 2\. Creating a virtual machine.
+
+![](https://raw.githubusercontent.com/haiku/website/master/static/files/new-vm-step-1.png)
+
+![](https://raw.githubusercontent.com/haiku/website/master/static/files/new-vm-step-2.png)
+ 
+![](https://raw.githubusercontent.com/haiku/website/master/static/files/new-vm-step-3.png)
```
Comment (waddlesplash) [inline content/guides/virtualizing/vmware-fusion.md]: These URLs need to match the other ones (see one of the merged files for this), they can't point to GitHub. Also, you can delete the added images, these links point to the existing ones.

File: content/guides/virtualizing/vmware-fusion.md (hunk 2 above)
Comment (johngarbi) [inline content/guides/virtualizing/vmware-fusion.md]: There isn't another way that I can find to link to another part of the document.

File: content/guides/virtualizing/vmware-fusion.md (hunk 1 above)
Comment (johngarbi) [inline content/guides/virtualizing/vmware-fusion.md]: By wrapping it with "<div class="box-info">content</div>"?

Comment (johngarbi) [inline content/guides/virtualizing/vmware-fusion.md]: By wrapping it with `<div class="box-info">content</div>`?

File: content/guides/virtualizing/vmware-fusion.md (hunk 5 above)
Comment (johngarbi) [inline content/guides/virtualizing/vmware-fusion.md]: The URLs are now relative. (like in the other files)

File: content/guides/virtualizing/vmware-fusion.md (hunk 2 above)
Comment (waddlesplash) [inline content/guides/virtualizing/vmware-fusion.md]: ?? Why can't the . be removed?

Comment (johngarbi) [inline content/guides/virtualizing/vmware-fusion.md]: Ohh. I thought you wanted the HTML removed.

File: content/guides/virtualizing/vmware-fusion.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+After downloading the ISO image, unzip the file and in the folder, you will see a .iso file. Under Virtual Machine Library, which you can use the shortcut <kbd>Shift</kbd>+<kbd>cmd</kbd>+<kbd>L</kbd>, click Add and then New.
+
+##### Step 2. Creating a virtual machine.
+
+![](/static/files/new-vm-step-1.png)
+
+![](/static/files/new-vm-step-2.png)
+ 
+![](/static/files/new-vm-step-3.png)
```
Comment (waddlesplash) [inline content/guides/virtualizing/vmware-fusion.md]: No /static/; Hugo will remove that.

File: content/guides/virtualizing/vmware-fusion.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+
+After downloading the ISO image, unzip the file and in the folder, you will see a .iso file. Under Virtual Machine Library, which you can use the shortcut <kbd>Shift</kbd>+<kbd>cmd</kbd>+<kbd>L</kbd>, click Add and then New.
+
+##### Step 2. Creating a virtual machine.
+
+![](files/new-vm-step-1.png)
+
+![](files/new-vm-step-2.png)
+ 
+![](files/new-vm-step-3.png)
```
Comment (waddlesplash) [inline content/guides/virtualizing/vmware-fusion.md]: Did you even look at the other files? You still need the first `/`.

**Pull request merged**